### PR TITLE
Poison Pill deprecation notice.

### PR DESCRIPTION
**Pull request by mshitrit**
Adding  deprecation notice in poison pill readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+# Deprecation Notice
+
+**July 28, 2022**
+
+**Poison Pill Operator** is deprecated in favor of [Self Node Remediation](https://github.com/medik8s/self-node-remediation),
+thus Poison Pill  is moving to a maintenance mode only and we are not taking PRs except for urgent bug fixes.
```
Comment (razo7) [inline README.md:6]: NIT: There are two spaces between `Pill` and `is` instead of one.

Comment (razo7) [reply README.md:6]: Do we have a support life cycle policy for PP? Meaning that we won't support PP from date x/y/z

Comment (slintes) [reply README.md:6]: this is upstream, no support at all 😉

**Pull request merged**